### DISCUSSION TITLE
Screen annotation feature like screenpen.

### DISCUSSION THREAD

**Discussion by enthusiva** [Gustash/Hyprshot#73] [Ideas]
It would be great to have a feature/mode that allows to draw/annotate on screen before taking screenshot.

**Comment by mrBars1k** [comment]
I fully agree with it! 
It would be the most ideal program for screenshots if they added the ability to draw.
For example, like in Lightshot on Windows.

It would also be convenient to automatically upload a screenshot to a server (for example imgur).
